index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Microverse Portfolio</title>
+  </head>
+  <body>
+    <section class="nav-section relative">
+      <nav class="w-full">
+        <div class="flex align-items-center h-4">
+          <div class="nav-items text-center">
+            <a href="#"><span class="fw-600">Portfolio</span></a>
+            <a href="#"><span class="fw-600">About</span></a>
+            <a href="#"><span class="fw-600">Contact</span></a>
+          </div>
+          <div class="nav-container">
+            <input class="checkbox" type="checkbox" />
+            <div class="menu-button">
+              <span class=""></span>
+              <span class=""></span>
+              <span class=""></span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </section>
+
+    <div class="hero-bg absolute"></div>
+
+    <section
+      class="hero-section relative flex flex-col align-items-center h-screen"
+    >
+      <div class="hero-container">
+        <div class="hero-content fw-700 leading-4 w-full text-center">
+          <h3 class="text-dark hero-title">
+            Hey there. I'm Alex. <br />
+            <span class="text-primary">I'm a software developer</span>
+          </h3>
```

Comment:
To give your app a good semantic score, kindly ensure you obey the header hierarchy rule😉 Use an `h1` for the main header of the page😉 and other headers will use the subsequent headers `h2, h3, h4, etc`. This makes your page understandable to screen readers👍. Kindly replace the h3 tag with a h1